component/src/main/java/org/wso2/grpc/EventServiceGrpc.java

```diff
@@ -0,0 +1,355 @@
+package org.wso2.grpc;
+
```

Comment:
If its GRPC generated class add a comment and also all files need to have license !!

Reply:
Is it okay to add wso2 licenses for autogenerated classes?

Reply:
yes